docs/source/development-guidelines.rst

```diff
@@ -0,0 +1,223 @@
+   $ pip install -r ./etc/requirements_blaze.txt 
+   $ pip install -r ./etc/requirements_talib.txt 
+
+Finally, you can build the C extensions by running:
+
+.. code-block:: bash
+
+   $ python setup.py build_ext --inplace
+
+To finish, make sure tests pass by running
+
+.. code-block:: bash
+
+   $ nosetests
+
+If you get an error running nosetests after setting up a fresh virtualenv, please try running
+
+.. code-block:: bash
+
+   # where zipline is the name of your virtualenv
+   $ deactivate Zipline
+   $ workon Zipline
+
+
+Development with Docker
+-----------------------
+
+If you want to work with zipline using a `Docker`__ container, you'll need to build the ``Dockerfile`` in the Zipline root directory, and then build ``Dockerfile-dev``. Instructions for building both containers can be found in ``Dockerfile`` and ``Dockerfile-dev``, respectively.
+
+__ https://docs.docker.com/get-started/
+
+
+Style Guide & Running Tests
+---------------------------
+
+We use `flake8`__ for checking style requirements and `nosetests`__ to run Zipline tests. Our `continuous integration`__ tools will run these commands.
+
+__ http://flake8.pycqa.org/en/latest/
+__ http://nose.readthedocs.io/en/latest/
+__ https://en.wikipedia.org/wiki/Continuous_integration
+
+Before submitting patches or pull requests, please ensure that your changes pass when running:
+
+.. code-block:: bash
+
+   $ flake8 Zipline tests
+
+In order to run tests locally, you'll need `TA-lib`__, which you can install on Linux by running:
+
+__ https://mrjbq7.github.io/ta-lib/install.html
+
+.. code-block:: bash
+
+   $ wget http://prdownloads.sourceforge.net/ta-lib/ta-lib-0.4.0-src.tar.gz
+   $ tar -xvzf ta-lib-0.4.0-src.tar.gz
+   $ cd ta-lib/
+   $ ./configure --prefix=/usr
+   $ make
+   $ sudo make install
+
+And for ``TA-lib`` on OS X you can just run:
+
+.. code-block:: bash
+
+   $ brew install ta-lib
+
+You should now be free to run tests:
+
+.. code-block:: bash
+		
+   $ nosetests
+
+
+Continuous Integration
+----------------------
+
+We use `Travis CI`__ for Linux-64 bit builds and `AppVeyor`__ for Windows-64 bit builds.
+
+.. note::
+
+   We do not currently have CI for OSX-64 bit builds, and do not support 32-bit builds.
+
+__ https://travis-ci.org/
+__ https://www.appveyor.com/
+
+
+Packaging
+---------
+To learn about how we build Zipline packages on `Anaconda`__ you can read `this`__ section in our release process notes.
+
+__ https://anaconda.org/
+__ http://www.zipline.io/release-process.html#uploading-conda-packages
+   
+Contributing to the Docs
+------------------------
+
+If you'd like to contribute to the documentation, you can navigate to ``docs/source/`` where each `reStructuredText`__ or ``.rst``, file is a separate section here on zipline.io. To add a section, create a new file called ``some-descriptive-name.rst`` and add ``some-descriptive-name`` to ``appendix.rst``. To edit a section, simply open up one of the existing files, make your changes, and save them.
```

Comment:
I have a little trouble parsing the first sentence here. Maybe:
"If you'd like to contribute to the documentation on zipline.io, you can navigate to ``docs/source/``, where each `reStructuredText`__ file (``.rst``) is a separate section there."